carts/serializers.py

```diff
@@ -0,0 +1,71 @@
+"""
+Serializers for carts models
+"""
+from django.utils.translation import gettext_lazy as _
+from rest_framework import serializers
+
+from carts.models import Cart, CartItem
+
+
+class CartSerializer(serializers.ModelSerializer):
+    """
+    Cart model serializer
+    """
+
+    def create(self, validated_data):
+        validated_data["created_by"] = validated_data["user"]
+        return Cart.objects.create(**validated_data)
+
+    class Meta:
+        """
+        Tells the fields to include in parsed/json object
+        """
+
+        # pylint: disable=protected-access, no-member
+        model = Cart
+        fields = "__all__"
```

Comment:
Are we doing any customizations or we need this serializer at this point?

Reply:
We needed it before today's changes because it assigned the `created_by` field but, now that field is being assigned in the `AuditTimeStamp` so we can remove this `create()` method for now. But we need the serializer as we are using it to serialize the cart object created.